packages/app-treasury/src/Overview/index.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright 2017-2019 @polkadot/app-democracy authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { AppProps, BareProps, I18nProps } from '@polkadot/ui-app/types';
+
+import '../index.css';
+
+import React from 'react';
+
+import Summary from './Summary';
+import Proposals from './Proposals';
+
+type Props = AppProps & BareProps & I18nProps;
+
+export default class Overview extends React.PureComponent<Props> {
+  render () {
+    return (
+      <>
+        <Summary />
+        {/*<Referendums />*/}
```

Comment:
I?sn't there the concept of "stuff to be voted on now"?

If so, once again, would look at the new-ish democracy, it uses a 2 Column layout to do these

Reply:
I don't know, it seems like things can be proposed and then approved or rejected so no second phase I don't think?